utils/time.ts

```diff
@@ -0,0 +1,71 @@
+import dayjs, { Dayjs } from 'dayjs';
+import utc from 'dayjs/plugin/utc'; // dependent on utc plugin
+import timezone from 'dayjs/plugin/timezone';
+import { ProgramView_events } from '../__generated__/ProgramView_events.graphql';
+import { ArrayElement } from './types';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+export const timeIsBetween = (
+  time: Dayjs,
+  start: string,
+  end: string,
+  date: string
+): boolean => {
+  const s = dayjs(`${start} ${date}`, 'HH:mm:ss YYYY-MM-DD');
+  const e = dayjs(`${end} ${date}`, 'HH:mm:ss YYYY-MM-DD');
+  const now = dayjs(time);
```

Comment:
could you call this something else? It's a bit confusing to me that it's called now when it's an argument :smile:

Reply:
Sure, it could be `start` and `end`
```suggestion
  const start = dayjs(`${start} ${date}`, 'HH:mm:ss YYYY-MM-DD');
  const end = dayjs(`${end} ${date}`, 'HH:mm:ss YYYY-MM-DD');
```